backend/src/apiserver/resource/resource_manager_test.go

```diff
@@ -1524,18 +1489,6 @@ func TestDeletePipelineVersion_FileError(t *testing.T) {
 	assert.True(t, ok)
 	pipelineStore.SetUUIDGenerator(util.NewFakeUUIDGeneratorOrFatal(FakeUUIDOne, nil))
 	manager.CreatePipelineVersion(pv)
-
-	// Switch to a bad object store
-	manager.objectStore = &FakeBadObjectStore{}
-
-	// Delete the above pipeline_version.
-	err = manager.DeletePipelineVersion(FakeUUIDOne)
-	assert.NotNil(t, err)
-
-	// Verify the version in deleting status.
-	version, err := manager.pipelineStore.GetPipelineVersionWithStatus(FakeUUIDOne, model.PipelineVersionDeleting)
-	assert.Nil(t, err)
-	assert.NotNil(t, version)
```

Comment:
it didn't jump out at me why this entire block is being deleted. Is there nothing here test-worthy if the object store storage goes away? Specifically wondering if it makes sense to keep the check of the deleting status.

Reply:
yeah, this specific test can probably remain, albeit the expected results are now the exact opposite: the RM should *not* throw an error if the ObjStore is bad, *should* be able to delete the pipelineversion, and the pipelineversion should *not* exist after performing the delete operation (essentially flip the asserts in L1533, L1537, and L1538)